### DISCUSSION TITLE
Apple Car千呼萬喚不出來

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS4719-C/Python-Community-GS4719-C#775] [General]
感覺大家關注電動車的重點不外乎就是電池續航和自動駕駛，在這方便我還是比較看好Tesla的電動車，畢竟不管在量產技術、能源技術、AI自動駕駛技術都領先各個車廠一大截。不知大家是怎沒想的？
https://ec.ltn.com.tw/article/breakingnews/4082220

**Comment by evanchou1204** [comment]
肯定推Tesla，馬斯克是外星人，感覺apple car要打敗Tesla難度頗大，只能看果粉的錢包夠不夠給力了。

**Comment by danwan1027** [comment]
雖然很多汽車廠商推出的自駕車已經可以比擬特斯拉，我個人認為特斯拉擁有自駕的數據庫遠大於其他車廠，在自駕這方面仍是其他品牌望塵莫及的。

**Comment by Pietra1226** [comment]
目前自駕車首選還是Tesla，而且他蒐集的數據佔市場大多數，慢慢地後來新興的公司就很難追上他們的數據量

**Comment by CPCpyt** [comment]
目前自駕車特斯拉的呼聲最高，不過非常期待日後自駕車能夠再有甚麼進一步的突破。

**Comment by ruthchen1020** [comment]
Tesla在汽車工藝及AI建模的技術上領先apple跟google很多，我覺得他們要追上電動車的步伐很難，畢竟Tesla是有火箭基礎的，對金屬的加工工藝水準很高。
google跟apple可能可以分機器人這塊大餅吧，做機器人比較簡單，但特斯拉因為之前的電動車AI識別技術很有底，所以很難超過了。

**Comment by StevenWanglolz** [comment]
瑪斯克是個瘋狗天才，所以特斯拉才會這麼成功! Apple還是乖乖生產手機吧

**Comment by swerfg03joe** [comment]
沒什麼實際意義的新聞，多半是用來炒作尚未推出的APPLE CAR，文章表示說要超越TESLA有限的自動駕駛技術，但實際上開上路手還是不能完全脫離方向盤，另外若是車用軟體完全綁死IOS系統等於強迫你只能用他們的產品，弔詭的是最近才想做汽車的公司，車用軟體能一次做好的可能性有多大?
